azavea/temperate
# Geospatial data ingest

## Overview

This PR adds some new models, with fixtures containing data derived from EPA Climate Damage & CDC NEPHT exports.

### Demo

Climate Assessment region w/ data:
![localhost_8100_admin_planit_data_climateassessmentregion_7_change_(iPad Pro)](https://user-images.githubusercontent.com/4432106/65653185-4aad8700-dfe2-11e9-8537-a0638d687be2.png)

County w/ data:
![localhost_8100_admin_planit_data_county_3222_change_(iPad Pro)](https://user-images.githubusercontent.com/4432106/65653186-4aad8700-dfe2-11e9-864b-78d052d584eb.png)


### Notes

 - Data for counties & states came from https://www.census.gov/geographies/mapping-files/time-series/geo/carto-boundary-file.html, which I then converted to GeoJSON. I used the highest resolution available, I suspect we may have to simplify this when serving it as GeoJSON as part of #1290 
 - We already have regions for the National Climate Assessment (stored in the poorly named `GeoRegion` model), but they are for version 3 of the report, and version 4 has updated the regions. The EPA Climate Damage datasets used the new regions, so I ingested them as part of this. I'll make a follow-on issue for migrating our existing `GeoRegion` uses to the new `ClimateAssessmentRegion` model.


## Testing Instructions

It's probably not worthwhile to rerun the ingest scripts since I'm committing the new fixtures.

 * `scripts/update`
 * Spot check the ingested data in the Django Admin


Closes #1268
